src/main/daos/Challenge.kt

```diff
@@ -220,7 +220,7 @@ object SubChallenges : BaseIntIdTable("sub_challenge") {
 data class ChallengeNamespace(
     val parentChallenge: Int?,
     val challengeSettings: ChallengeSettingNamespace,
-    val subChallenges: List<Pair<Int, SubChallengeType>>,
+    val subChallenges: List<Pair<Int, SubChallengeType>>?,
```

Comment:
val subChallenges: List<Pair<Int, SubChallengeType>>? = listOf()

Reply:
are you saying to make this a default value for the arg?

Reply:
yes